storage/src/adb/operation.rs

```diff
@@ -123,7 +123,7 @@ impl<K: Array, V: Array> Read for Operation<K, V> {
                 // Check that the value is all zeroes
                 for _ in 0..V::SIZE {
                     if buf.get_u8() != 0 {
-                        return Err(CodecError::Invalid("Operation", "Delete value non-zero"));
+                        return Err(CodecError::Invalid("operation", "delete value non-zero"));
```

Comment:
We've used a full path for "Invalid" before...tbh we may be able to pull this automatically with some rust built-in: https://github.com/commonwarexyz/monorepo/blob/127460975f99911c29d5d9da5df21e1ab1342b1d/consensus/src/aggregation/types.rs#L286

Reply:
Updated to `"storage::adb::Operation"`